synapse/config/tls.py

```diff
@@ -343,7 +369,7 @@ def generate_config_section(
             # ACME support is disabled by default. Uncomment the following line
             # (and tls_certificate_path and tls_private_key_path above) to enable it.
             #
-            #enabled: true
+            %(acme_enabled)senabled: true
```

Comment:
Can we do:

```suggestion
            enabled: %(acme_enabled)s
```

Instead? Then we can have the below always uncommented